src/deckgl-layers/layer-utils/cluster-utils.js

```diff
@@ -41,11 +41,9 @@ export function getGeoJSON(data, getPosition) {
 const clusterResolver = ({clusterRadius}) => `${clusterRadius}`;
 
 const getClusterer = memoize(({clusterRadius, geoJSON}) => {
-  return supercluster({
+  return new Supercluster({
     maxZoom: 20,
     radius: clusterRadius,
-    initial: () => ({points: []}),
```

Comment:
this is a initial state for hovered poitns, and is used to calculate hover state. Is there a new prop that replacing it?

Reply:
v6.0.0 [removed the initial](https://github.com/mapbox/supercluster/releases/tag/v6.0.0) argument. It now uses the first item as a basis. The hovered points should still work, no?

Reply:
According to docs:
```
In most cases, you can simply remove the initial option and things will work. There are some exceptions where you'd need to update your map function to make up for lack of initial, but they should be rare.
```
We should then update the map function, see comment below

Reply:
originally initial function add an empty array of points. then the reduce function will push props.data to the `points` array. 

We need the property `points` in each cluster to store all the points that it contains. It is used to render hover info. Please make sure they are propogated

Reply:
It pulls the `points` property from the geoJSON object: https://github.com/keplergl/kepler.gl/blob/abe8e9b8e20d4f3b430c2c3c7d7350c838d49907/src/deckgl-layers/layer-utils/cluster-utils.js#L30

As far as I can tell it works correctly? I can see it changing the color of the hovered icon:

![kepler-cluster-hover](https://user-images.githubusercontent.com/931368/60051117-46e72d00-96a0-11e9-92fc-9354adca22c9.gif)

If I'm missing something, how can I reproduce?

Reply:
it should show total points correctly
![Screen Shot 2019-06-24 at 2 05 56 PM](https://user-images.githubusercontent.com/3605556/60052086-498a5800-9689-11e9-8dc9-c66529ce3580.png)

Reply:
You should keep the `map: props => props.data` function